src/processing/dom/base-dom-adapter.js

```diff
@@ -3,7 +3,7 @@
 // Do not use any browser or node-specific API!
 // -------------------------------------------------------------
 
-const IFRAME_FLAG_TAGS = ['a', 'form'];
+const IFRAME_FLAG_TAGS = ['a', 'form', 'area'];
```

Comment:
No, I don't think so because we don't need to append the iframe flag to base element.